remotespark/livyclientlib/livysession.py

```diff
@@ -3,7 +3,7 @@
 
 import textwrap
 from time import sleep, time
-
+from IPython import get_ipython
```

Comment:
Remove any mention to IPython from within Livy classes. They should be agnostic to the view.